modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyBranch.jsx

```diff
@@ -200,8 +226,22 @@ function VocabularyBranch(props) {
       }
 
       {/* Term name */}
-      <Typography onClick={() => loadTerm(id, path)}
-                  className={classes.infoName + (focused ? (" " + classes.focusedTermName) : " ")}>
+      <Typography onClick={(evt) => {(evt.target.type != "checkbox") && loadTerm(id, path)}}
+                  className={classes.infoName + (focused ? (" " + classes.focusedTermName) : " ")}
+                  component="div">
+        {/* Browser term select tools */}
+	    { addCheckbox &&
+	      <Checkbox
+	        color="secondary"
+	        checked={selectedPaths.includes(path)}
+	        onClick={(evt) => {event.stopPropagation(); onTermSelect(evt, path, name);}}
```

Comment:
Who is `event`, shouldn't it have been `evt.stopPropagation()`? But isn't it redundant with the first line of `onTermSelect`? I would also rename the function to `onSelectionChanged` to cover both select and unselect, and to simplify the code I would remove the path and name params which should be already available in the scope of the function.

```suggestion
	        onClick={onSelectionChanged}
```

Reply:
Addressed in 1b7e199